src/components/navigation-bar/navigation-bar.stories.mdx

```diff
@@ -24,4 +41,19 @@ import NavigationBar from './navigation-bar.component';
   </Story>
 </Preview> 
 
+###  isLoading
+<Preview>
+  <Story name="dark is loading" parameters={{ info: { disable: true }}}>
+    <div> 
+      <div> If `isLoading` is `true` a content will not be visible </div>
```

Comment:
```suggestion
      <div> If `isLoading` is `true` the content will not be visible </div>
```

Reply:
These comments should be outside of the `<Preview>`